docs/services/frame-system/_index.md

```diff
@@ -0,0 +1,173 @@
+| `Geometry`  | **Required** | Default: `none`. Collision geometries for defining bounds in the environment of the robot. <br> Types: `Sphere`, `Box`, and `Capsule`. |
+
+{{% alert title="Caution" color="caution" %}}
+
+`Types` are offered in `Orientation` for ease of configuration, but the [orientation vector](/internals/orientation-vector/) type that is always stored and returned by the Frame System is `"Orientation Vector Radians"`.
+`"Degrees"` and `"Quaternion"` will be converted to `"Radians"`.
+
+{{% /alert %}}
+
+{{% alert title="Tip" color="tip" %}}
+
+Viam's coordinate system considers `+X` to be forward, `+Y` to be left, and `+Z` to be up.
+
+{{% /alert %}}
+
+For more information, see these two examples:
+
+- [A component attached to a static surface](/component-on-static)
+- [A component attached to another, dynamic, component](/component-on-dynamic)
+
+## Building the Frame System
+
+Once you configure your robot and run `viam-server`, your robot builds a tree of reference frames with the `world` as the root node.
+
+A [topologically-sorted list](https://en.wikipedia.org/wiki/Topological_sorting) of the generated reference frames is printed by the server and can be seen in the logs at `--debug` level:
+
+![an example of a logged frame system](img/frame_sys_log_example.png)
+
+`viam-server` regenerates this tree in the process of [reconfiguration](/manage/fleet-management/#configurationlogging).
+
+Let's consider the example of a [component attached to a dynamic component](/component-attached-to-a-dynamic-component): a robotic arm, `A`, attached to a gantry, `G`, which in turn is fixed in place at a point on the `World` of a table.
+
+The resulting tree of reference frames looks like:
+
+![reference frame tree](img/frame_tree.png)
+
+`viam-server` builds the connections in this tree by looking at the `"frame"` portion of each component in the robot's configuration and defining *two* reference frames for each component:
+
+1. One with the name of the component, representing the actuator or final link in the component's kinematic chain: like `"A"` as the end of an arm.
+2. Another representing the origin of the component, defined with the component's name and the suffix *"_origin"*.
+
+## Accessing the Frame System
+
+The [Robot API](https://github.com/viamrobotics/api/blob/main/proto/viam/robot/v1/robot.proto) supplies two methods to interact with the Frame System through gRPC calls:
+
+1. `TransformPose`: Transforms a pose measured in one reference frame to the same pose as it would have been measured in another.
+2. `FrameSystemConfig`: Returns a topologically sorted list of all the reference frames monitored by the frame system.
+Any [supplemental transforms](#handling-motion-with-supplemental-transforms) are also merged into the tree, topologically sorted, and returned.
+
+## Supplemental Transforms
```

Comment:
I need to read over this section more thoroughly, there is a decent amount of information here that feels outdated and I want to double-check that.

Reply:
@biotinker and I are going to talk about this together during our morning 1:1, so we should have some updates for you shortly.